etl/load.py

```diff
@@ -51,14 +52,52 @@ def _generate_upsert_sql(mon_loc):
         f"MERGE INTO GW_DATA_PORTAL.WELL_REGISTRY_STG a "
         f"USING (SELECT '{mon_loc['AGENCY_CD']}' AGENCY_CD, '{mon_loc['SITE_NO']}' "
         f"SITE_NO FROM DUAL) b ON (a.AGENCY_CD = b.AGENCY_CD AND a.SITE_NO = b.SITE_NO) "
-        f"WHEN MATCHED THEN UPDATE SET {update_query} WHEN NOT MATCHED THEN INSERT ({all_columns}) VALUES ({all_values})"
+        f"WHEN MATCHED THEN UPDATE SET {update_query} "
+        f"WHEN NOT MATCHED THEN INSERT ({all_columns}) VALUES ({all_values})"
     )
     return statement
 
 
+def _generate_upsert_pgsql(mon_loc):
+    """
+    Generate SQL to insert/update for PostGIS
+    """
+    mon_loc_db = [(k, _manipulate_values(v, k in TIME_COLUMNS)) for k, v in mon_loc.items()]
+    all_columns = ','.join(col for (col, _) in mon_loc_db)
+    all_columns += ',geom'
+    all_values = ','.join(value for (_, value) in mon_loc_db)
+    all_values += f", ST_MakePoint({mon_loc['DEC_LAT_VA']},{mon_loc['DEC_LONG_VA']})"
+    update_query = ','.join(f"{k}={v}" for (k, v) in mon_loc_db if k not in ['AGENCY_CD', 'SITE_NO'])
+
+    statement = (
+        f"INSERT INTO GW_DATA_PORTAL.WELL_REGISTRY_MAIN a ({all_columns}) VALUES ({all_values}) "
```

Comment:
Looks to be making a table alias 'a' for GW_DATA_PORTAL.WELL_REGISTRY_MAIN
do not see that it is used, could be removed

Reply:
yes, it could be removed.